docs/UserGuide.md

```diff
@@ -148,6 +142,31 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+### Uploading cards : `upload`
+
+Imports cards to LingoGO! using a CSV file.
+
+Format: `upload CSV_FILE_PATH`
+
+* The file path can be absolute or relative to the location of the LingoGO! file.
+* The CSV file must have 2 columns. The first column is for English and the second column is for Foreign language.
+  ![sample CSV file](images/SampleCSVFile.png)
+
+Examples:
+* `upload ./dictionary.csv` will load all cards stored in the dictionary.csv to LingoGO!
+
+### Downloading cards : `download`
+
+Exports cards from LingoGO! to a CSV file.
+
+Format: `download`
+
+* The exported file will be added to `data` folder in a CSV format.
+* The CSV file will have 2 columns. The first column is for English and the second column is for Foreign language.
+
+Examples:
+* `download` will load all cards stored in LingoGO! to the dictionary.csv.
```

Comment:
Delete the examples section here since the command does not require any arguments.